.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "nuklear-sys/nuklear-c/nuklear"]
 	path = nuklear-sys/nuklear-c/nuklear
-	url = https://github.com/vurtun/nuklear
+	url = ../../Immediate-Mode-UI/Nuklear
```

Comment:
Hm. The repo was moved to https://github.com/Immediate-Mode-UI/Nuklear - why can't we use this updated address to fetch the code?

Reply:
Assuming you're asking why the path is relative.

An absolute path such as 'https://github.com/Immediate-Mode-UI/Nuklear' forces git to fetch the submodule via the https protocol. For worlflows where ssh protocol is preferred, this means having to manually modify the submodule in the repo.
When a relative path is used, git will use the same protocol for the submodule as the one that was used for cloning the repo, https or ssh.

This StackOverflow answer has a somewhat longer explanation: https://stackoverflow.com/a/44630028/113452.

Reply:
@snuk182 do you have any additional thoughts and/or verification that you think would be good to do before this can be merged?